src/components/_uswds.njk

```diff
@@ -7,5 +7,13 @@
   <body>
     {% block body %}{{ yield }}{% endblock %}
     <script src="{{ uswds.path }}/js/uswds.min.js"></script>
+    <script>
+    window.addEventListener('load', function() {
+      document.body.addEventListener('submit', function(e) {
+        // https://github.com/18F/web-design-standards/issues/2101
+        e.preventDefault();
+      }, true);
+    }, false);
+    </script>
```

Comment:
I ended up using an inline script instead of creating a JS file because the static assets directory for our fractal site is `dist`, and I didn't want to pollute it with static files that are unrelated to core USWDS.